stubs/mailchimp3/__init__.pyi

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Dict, Optional, Text
+
+class MailChimp(object):
+    lists: Lists = ...
+
+class Lists:
+    members: ListMembers = ...
+
+class ListMembers:
```

Comment:
Can we limit the `Any` here?

Reply:
I actually tried doing so but you can see [here](https://github.com/VingtCinq/python-mailchimp/blob/122b772cc36206458c17c218d2b5170db350e383/mailchimp3/entities/listmembers.py#L37) that a response object is returned. If we follow how the response here is obtained, we figure out that it the JSON response returned and it can be of type Any. Thanks!